pkg/bundled/cloudconfigs/00_rootfs.yaml

```diff
@@ -53,17 +53,25 @@ stages:
           /var/lib/wicked
           /var/lib/kairos
           /var/log
-          /var/run/cilium
```

Comment:
cilium writes some `.sock` files there from what I can tell by googling this path. Are you sure this is ok?